src/emulator/pubsubEmulator.ts

```diff
@@ -62,7 +70,21 @@ export class PubsubEmulator implements EmulatorInstance {
   }
 
   async stop(): Promise<void> {
-    await downloadableEmulators.stop(Emulators.PUBSUB);
+    try {
+      await downloadableEmulators.stop(Emulators.PUBSUB);
+    } catch (e: unknown) {
+      this.logger.logLabeled("DEBUG", "pubsub", JSON.stringify(e));
+      if (process.platform !== "win32") {
+        exec(PUBSUB_KILL_COMMAND, (err, stdout) => {
```

Comment:
I just noticed: exec isn't synchronous, should this be wrapped in a Promise and awaited, or use execSync?

Reply:
This can be a fire-and-forget so long as Node doesn't terminate before executing it. In that case I think we'll just be missing the output. At any rate I think wrapping in a promise like this should do the trick.